src/content/core/components/namespaces.mdx

```diff
@@ -31,18 +46,27 @@ To share a namespace go to the Okteto dashboard, select the namespace you want t
 
 <p align="center"><Image src={require('@site/static/img/share.png').default} alt="share a namespace" width="850" /></p>
 
-In the `Share` dialog, type the email address of the team members you want to share this namespace with. Once you press save, Okteto dashboardll update the necessary permissions and notify your teammates via email.
+In the `Share` dialog, type the email address of the team members you want to share this namespace with. Once you press save, Okteto dashboard will update the necessary permissions and notify your teammates via email.
 
-You can sleep your namespace from the command line with `okteto namespace sleep [name]`.
 
-When you want to wake your namespace, you can use the command `okteto namespace wake [name]`.
+## Transfer your namespace
+
+Okteto administrators have the ability to transfer non-default namespaces between users. This process can be achieved through the Okteto dashboard.
+
+To transfer a namespace to a new owner, go to the Okteto dashboard -> Admin -> Namespaces then locate the namespace that you would like to transfer. Click on the three dots `...` to display the namespace menu. From the namespace menu select Change Owner.
 
-You can sleep your namespace from Okteto dashboardself by clicking on Sleep dialog.
+<p align="center"><Image src={require('@site/static/img/namespace-change-owner.png').default} alt="share a namespace" width="650" /></p>
+
+When the Change Owner modal appears, type the email address of the team member who you'd like the transfer the namespace to. Once you press confirm, ownership of that namespace will be immediately transferred.
```

Comment:
I would name the modal `Transfer Ownership` since it's the actual title of the modal, rather than the option name in the dropdown menu.
```suggestion
When the Transfer Ownership modal appears, type the email address of the team member who you'd like the transfer the namespace to. Once you press confirm, ownership of that namespace will be immediately transferred.
```

Reply:
I'm generally supportive of this suggested change. Why did we opt for two different names though between the menu and the modal? Wouldn't it make sense to keep it the same?

Not a blocking question/suggestion, just curious and to know if we could consider an update in future releases 😄

Reply:
It looks like there was a discrepancy in the design file. The original said "Change Owner" and an update said "Transfer Ownership." My bad I should have caught that.
@bgonp is to to late to change the modal title to "Change Owner" to match the dropdown menu?

Reply:
The 1.17 cut has been done but we can create a quick backport for this thing.
Just a note: AFAIK internally both backend and fronted use the "transfer ownership" term. Do we want to take the "Change Owner" as the _official_ name everywhere or "Transfer Ownership"?

Reply:
I believe "Transfer Ownership" is concise and more accurately describes the feature.
Could we standardize on that? I'll go ahead and update all documentation.